appveyor.yml

```diff
@@ -12,7 +12,7 @@ environment:
 build_script:
   - sbt version # dummy step, a build_script step is required by appveyor.
 test_script:
-  - sbt testsJVM/test
+  - sbt testOnlyJVM
```

Comment:
Do the JS tests also pass on Windows? We can run `testAll`  on Appveyor, it would make sure the Node.js IO facades function as expected on Windows.

Reply:
Alas, 9 failures still. In scope of this PR or a separate one?

Reply:
This PR already greatly improves the state of the art so we can separate it if you prefer, up to you.

Reply:
We already split once - If we can have the full test suite pass on Appveyor rather than most of it we can have a real base line and regression prevention. That's probably worth getting it in here.